tribits/core/package_arch/TribitsExternalPackageWriteConfigFile.cmake

```diff
@@ -567,13 +567,19 @@ function(tribits_external_package_get_libname_from_full_lib_path  full_lib_path
     if (beginsWithLib)
       string(SUBSTRING "${full_libname}" 3 -1 libname)
     else()
-      set(libname "${full_libname}")
+      # Must be a framework dir with extension .framework
+      get_filename_component(last_ext "${full_lib_path}" LAST_EXT)
+      if (last_ext  STREQUAL ".framework")
```

Comment:
Hello @KyleFromKitware, here is where a comment in a commit is associated with the PR and therefore will be displayed with the PR.  This is where the commit https://github.com/bartlettroscoe/TriBITS/commit/2a6c1ceb222283de04c181a0a1db3d5a126d0fbc#commitcomment-77219732 would go.